python/src/trezorlib/tools.py

```diff
@@ -368,6 +369,7 @@ class Deprecated(value.__class__):
     else:
         # - for other types (we assume it's MessageType so a user-defined class)
         # assign __class__ directly
+        value = copy.copy(value)
```

Comment:
just from curiosity - why a shallow copy is needed here? (maybe add a comment would help)

Reply:
well because if you don't make a copy, you change the class of the passed-in value :) this *should not* matter in theory, but in practice in the test suite we do filtering and store the raw responses for later checking, and mutating the response messes up that mechanism
I'll add a comment.

Reply:
improved in f99c9f46b044

note to self: it doesn't belong in the same fixup so i'll have to split it out before merging